recipes/nibabel/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: nibabel
+  version: "0.11"
+
+source:
+    git_url: https://github.com/nipy/nibabel.git
+    git_tag: 2.0.2
+
+requirements:
+  build:
+    - python
+    - numpy
+  run:
+    - python
+    - numpy
```

Comment:
Sure, what I meant was is there compiled C/C++ or other languages that build libraries that link to NumPy libraries. It doesn't appear that is the case from the code I have seen. Is that a correct statement?